utils/utils.go

```diff
@@ -222,3 +226,128 @@ func GetPrintableString(v *string, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+// Encrypt data with AES-128 encryption algorithm and returns the encrypted data and the AES key encrypted with RSA
+func Encrypt(data []byte, pub *rsa.PublicKey) (string, string, error) {
+	initVector := []byte("\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00")
+
+	randomKey := make([]byte, 16)
+	_, err := rand.Read(randomKey)
+	if err != nil {
+		return "", "", errors.WrapErrorAction("generating", "random key", nil, err)
+	}
+	//Encrypt blobJSON with AES using random key(CBC mode, PKCS7 padding, 0 IV) and convert to base 64 to get encrypted_data
+	cipherBlock, err := aes.NewCipher(randomKey)
+	if err != nil {
+		return "", "", errors.WrapErrorAction(logutils.ActionCreate, "AES cipher block", nil, err)
+	}
+
+	paddedData, err := PKCS7Padding(data, cipherBlock.BlockSize())
+	if err != nil {
+		return "", "", errors.WrapErrorAction("padding", logutils.TypeString, nil, err)
+	}
+	cipherText := make([]byte, len(paddedData))
+	mode := cipher.NewCBCEncrypter(cipherBlock, initVector)
```

Comment:
Given known vulnerabilities of CBC mode, it is probably best to use GCM mode if possible.